CRM/Contact/Form/Search.php

```diff
@@ -902,4 +902,22 @@ public function getTitle() {
     return ts('Search');
   }
 
+  /**
+   * Do form value conversions based on form metadata.
+   */
+  public function convertFormValues() {
+    $metadata = $this->getSearchFieldMetadata();
+    foreach ($metadata as $fields) {
+      foreach ($fields as $field => $spec) {
+        if ($spec['type'] == CRM_Utils_Type::T_DATE) {
+          if (isset($this->_formValues[$field . '_low'])) {
```

Comment:
@eileenmcnaughton why only convert low? why not `_high` as well

Reply:
only because it's still incomplete

Reply:
@seamuslee001 fix pushed